dashboard/test/controllers/api/v1/projects/public_gallery_controller_test.rb

```diff
@@ -115,7 +115,7 @@ class Api::V1::Projects::PublicGalleryControllerTest < ActionController::TestCas
     project_row = categories_list['applab'].first
     assert_equal 'Charlies App', project_row['name']
     assert_equal 'STUB_CHANNEL_ID-1234', project_row['channel']
-    assert_equal 'https://studio.code.org/charlies_thumbnail.png', project_row['thumbnailUrl']
+    assert_equal '/v3/files-public/charlies_thumbnail.png', project_row['thumbnailUrl']
```

Comment:
Thanks for suggesting this, Will. Embarrassingly, my cache headers were not getting set!